src/ts/forwarders/forwarder.ts

```diff
@@ -116,34 +103,23 @@ export abstract class Forwarder<TParams extends Partial<MinimalTx>> {
    * @param tx ProxyAccountCallData or RelayCallData
    */
   public async signAndEncodeMetaTransaction(tx: TParams): Promise<MinimalTx> {
```

Comment:
So the point here was to be able to call 1 function and be able to deploy or send tx. I think it would be nice to maintain that - eg have a optional to.

Reply:
We have another ticket: #38 so maybe we can re-write all of these as part of that

Reply:
we could do it in new ticket; e.g. to simplify API. I have a feeling we could just rename it to signMetaTransaction() and re-name the internal function (e.g. to authoriseMetaTx).

Reply:
the "target" thing could potentially be addressed here.